pkg/services/ngalert/api/authorization.go

```diff
@@ -179,6 +179,13 @@ func (api *API) authorize(method, path string) web.Handler {
 		http.MethodPost + "/api/v1/ngalert/admin_config",
 		http.MethodGet + "/api/v1/ngalert/alertmanagers":
 		return middleware.ReqOrgAdmin
+
+	// Grafana-only Provisioning Read Paths
+	case http.MethodGet + "/api/provisioning/policies":
+		return middleware.ReqSignedIn
+
+	case http.MethodPost + "/api/provisioning/policies":
+		return middleware.ReqEditorRole
```

Comment:
I wonder if we can add FGAC actions to these endpoints. 
There seem to be provisioning APIs that are already covered by this https://github.com/grafana/grafana/blob/a5e4a533fa18d014a28f18630a450c36a67e2716/pkg/api/accesscontrol.go#L13

https://github.com/grafana/grafana/blob/572ca553b6aaef8c3de57fca62432f35d1be84aa/pkg/api/api.go#L509-L512

Reply:
I'd love to have dedicated provisioning roles. I'd envision users being able to create a provisioning-only API key to be used for their CI pipelines, long term. This is something we haven't fully settled on.

The particular difference here is that the actions linked above are only for "reloading" objects provisioned through another method - i.e. telling Grafana to re-read the YAML files off disk. The routes added by this PR are for directly creating/updating objects, which is more in-line with existing permissions. When we add file-based provisioning, we'll likely add a reload route at that point. This reload action feels distinctly different from direct creation too me.

Unrelated, but looks like our URL is slightly different from what the other routes are using. Will adjust.